### DISCUSSION TITLE
SD card?

### DISCUSSION THREAD

**Discussion by tuFFo126** [UzixLS/zx-sizif-512#56] [General]
What is the maximum size of SD cards supported in Sizif?

**Comment by UzixLS** [comment]
There is no limitations by Sizif itself - any 3.3V SD card with SPI interface should work.
But there may be limitations in ESXDOS, not sure what they are.
Personally I used cards up to 32 Gb wihout any problems.

**Reply by tuFFo126** [reply]
Thanks for the information.  I bought a 128GB SD card for the phone and checked how it works with Sizif.  When I directly copied the esxDOS files to it, the computer detected the card but did not find the esxDOS system.  The card had to be reformatted, but first a new "msdos" partition table had to be created, then after re-uploading the esxDOS files to the large 128GB partition, it caused sizif to boot into esxDOS.  So it's more about how to prepare the SD card.  In +3e mode, after issuing the "cat tab" command, cards are visible as a maximum of 8GB (65535/2/128),  even the larger ones, we simply won't use the entire capacity.
